package.json

```diff
@@ -39,28 +39,28 @@
     "package.json"
   ],
   "dependencies": {
-    "firebase": "^3 || ^4",
-    "jsdom": ">3"
+    "firebase": "*",
```

Comment:
Could you specify a version or version range in all the places you're using `"*"` (I see three, there may be a few more)

Reply:
Yeah, good point. I'll set that up.